husky_gazebo/CMakeLists.txt

```diff
@@ -22,9 +22,9 @@ install(
 
 if (CATKIN_ENABLE_TESTING)
   find_package(roslaunch REQUIRED)
-  roslaunch_add_file_check(launch/husky_empty_world.launch)
+  roslaunch_add_file_check(launch/empty_world.launch)
```

Comment:
If there's any possibility of documentation sticking around that might refer to the old name, you could drop a replacement file in the old location that `<include>`s the new one but is marked as deprecated:

http://wiki.ros.org/roslaunch/XML/launch#line-2

Reply:
I don't expect this to be an issue; I've got the necessary changes made to the Noetic docs and a separate merge-request to update that.  The older Kinetic and Melodic versions aren't being changed, so the older docs for those versions can remain as-is.